components/atoms/Checkbox/checkbox.tsx

```diff
@@ -9,31 +9,34 @@ interface CheckboxProps extends React.ComponentPropsWithoutRef<typeof CheckboxPr
 }
 
 const Checkbox = React.forwardRef<React.ElementRef<typeof CheckboxPrimitive.Root>, CheckboxProps>(
-  ({ className, label, id = "checkbox", ...props }, ref) => (
-    <div className="flex items-center">
-      <CheckboxPrimitive.Root
-        ref={ref}
-        className={clsx(
-          "peer h-4 w-4 shrink-0 rounded-[4px] cursor-pointer bg-white border border-light-slate-8 hover:border-orange-500 focus:outline-none disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:border-orange-500 data-[state=checked]:bg-orange-500",
-          className
-        )}
-        {...props}
-        id={id}
-      >
-        <CheckboxPrimitive.Indicator className={clsx("flex items-center justify-center text-white")}>
-          <FiCheck className="w-full h-full" />
-        </CheckboxPrimitive.Indicator>
-      </CheckboxPrimitive.Root>
-      {label && (
-        <label
-          htmlFor={id}
-          className="ml-3 text-sm font-medium leading-none cursor-pointer text-light-slate-12 peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
+  ({ className, label, id, ...props }, ref) => {
+    id = id ? id : label?.replaceAll(" ", "_").toLowerCase();
+    return (
+      <div className="flex items-center">
+        <CheckboxPrimitive.Root
+          ref={ref}
+          className={clsx(
+            "peer h-4 w-4 shrink-0 rounded-[4px] cursor-pointer bg-white border border-light-slate-8 hover:border-orange-500 focus:outline-none disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:border-orange-500 data-[state=checked]:bg-orange-500",
```

Comment:
```suggestion
            "peer h-4 w-4 shrink-0 rounded cursor-pointer bg-white border border-light-slate-8 hover:border-orange-500 focus:outline-none disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:border-orange-500 data-[state=checked]:bg-orange-500",
```

Reply:
Thank you for the suggestion did not notice this, @NsdHSO. As  it was already in the main design: 

https://github.com/open-sauced/insights/blob/49dff5b5092acce1f00102c88d612240a261f451/components/atoms/Checkbox/checkbox.tsx#L11-L22